+++ src/src/tool_xattr.h	2025-02-12 14:39:46.197217955 +0100
@@ -28,11 +28,6 @@
 #ifdef HAVE_FSETXATTR
 #  include <sys/xattr.h> /* header from libc, not from libattr */
 #  define USE_XATTR
-#elif (defined(__FreeBSD_version) && (__FreeBSD_version > 500000)) || \
-      defined(__MidnightBSD_version)
-#  include <sys/types.h>
-#  include <sys/extattr.h>
-#  define USE_XATTR
 #endif
 
 #ifdef USE_XATTR
